compiler_opt/rl/inlining/imitation_learning_runner.py

```diff
@@ -37,7 +37,9 @@ def main(_):
   logging.info(gin.config_str())
 
   generate_bc_trajectories_lib.gen_trajectories(
-      callable_policies=[imitation_learning_config.greedy_policy],
+      # set callable policies here directly since callables can not be
+      # gin configured when they are pickled
```

Comment:
Extremely minor nit: Comments are usually full sentences, so a capital and a period.